### PR TITLE
switch to GitHub actions

### PR DESCRIPTION
**Pull request by dlockhart**
@imcdonald-d2l & team opted to stick with the incremental-release method for this repo.

### CODE REVIEW

File: wct.conf.json [hunk 1]
```diff
@@ -10,27 +10,17 @@
...
 				{
 					"browserName": "chrome",
-					"platform": "OS X 10.12",
-					"version": ""
-				},
-				{
-					"browserName": "chrome",
-					"platform": "Windows 10",
-					"version": ""
-				},
```
Comment (dlockhart) [inline wct.conf.json:20]: Typically we don't bother with running tests for Chrome & Firefox on both Windows & macOS -- this should speed up your builds.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: CI
+on: pull_request
+jobs:
+  test:
+    timeout-minutes: 15
```
Comment (dlockhart) [inline .github/workflows/ci.yml:5]: Started at 5 minutes, increase to 10 and it passed in 9:58 so I've increased it again to 15 minutes.

**Review by imcdonald-d2l — APPROVED** [review]
Thanks! 🙂

**Review by brissons — APPROVED** [review]

**Review by sarahboyd — APPROVED** [review]
Thanks for doing this Dave!

**Pull request merged**